stream-chat-android-client/src/test/java/io/getstream/chat/android/client/call/CallTestUtils.kt

```diff
@@ -0,0 +1,5 @@
+package io.getstream.chat.android.client.call
+
+internal fun <T : Any> callFrom(valueProvider: () -> T): Call<T> = WrappedValueCall(valueProvider())
+
+internal fun <T : Any> T.asCall(): Call<T> = WrappedValueCall(this)
```

Comment:
These feel a bit like overkill for their current usages, but they can stay if you like them 😄

Reply:
I think it will be convenient for tests. When I faced that we hadn't anything like this in tests I was wondered. Because Call is wide spread thing in LLC